filters/drop-shadow/src/DropShadowFilter.js

```diff
@@ -30,14 +32,11 @@ export default class DropShadowFilter extends PIXI.Filter {
 
     apply(filterManager, input, output) {
         const target = filterManager.getRenderTarget();
-        target.transform = new PIXI.Matrix();
-        target.transform.translate(
-            this.distance * Math.cos(this.angle),
-            this.distance * Math.sin(this.angle)
-        );
+
+        target.transform = this.targetTransform;
         this.tintFilter.apply(filterManager, input, target, true);
         this.blurFilter.apply(filterManager, target, output);
-        super.apply(filterManager, input, output);
+        filterManager.applyFilter(this, input, output);
```

Comment:
@finscn why this change? Also with #103, we should probably pass `clear` to `applyFilter`

Reply:
Fixed. And many filters use `filterManager.applyFilter(this, input, output, clear)`.
And  filterManager.applyFilter is faster than `Filter.prototype.apply.call(...)`.
`super.apply(filterManager, input, output);` will became Filter.prototype.apply.call after built.

And  In `Filter.prototype.apply`  , there is `filterManager.applyFilter(...)`